kolibri_instant_schools_plugin/assets/src/views/sign-up-page/index.vue

```diff
@@ -75,22 +74,45 @@
         v-model="confirmedPassword"
       />
 
-      <ui-select
-        :name="$tr('selectFacility')"
-        :placeholder="$tr('selectFacility')"
-        :label="$tr('facility')"
-        :value="selectedFacility"
-        :options="facilityList"
-        :invalid="facilityIsInvalid"
-        :error="facilityIsInvalidText"
-        @blur="facilityBlurred = true"
-        @input="updateSelection"
-      />
+      <!-- ToS modal only shows up if the box empty prior to clicking -->
+      <!-- Using UI Checkbox for blur events and v-model -->
+      <label class="terms-agreement">
```

Comment:
More margin between this label and the textbox?

Reply:
Right now it's got the same margin as every other item in the form - did it that way for consistency. Textbox looks further away because we left space for the error message. Once the messages are up, it looks like this: 
![image](https://user-images.githubusercontent.com/9877852/32357794-34a995e0-bffd-11e7-8c7a-e1f130ed7c88.png)

You think having more space is appropriate? How much more?

Reply:
A margin-bottom of 40px for the re-enter password textbox? Also a margin-bottom of 8px for the .terms-agreement-checkbox?
![localhost_8000_user_ 2](https://user-images.githubusercontent.com/7193975/32385870-ad8221a6-c07c-11e7-9b7a-424bb52cbba8.png)

Reply:
How do you feel about 36 rather than 40? That's distance between the top of the input from the div that wraps it + height of the bottom margin for every textbox, so it'd look consistent with the rest of the fields.

Mind explaining why you'd like the terms to further from the rest of the form?

I understand the reasoning behind wanting to group the terms prompt and the checkbox, but they're already closer to each other than everything else on the page. 

![image](https://user-images.githubusercontent.com/9877852/32395131-23237d68-c09d-11e7-8e09-83a36526e50b.png)

Reply:
36 works. It just seemed a bit crowded to me.

Reply:
![space](https://user-images.githubusercontent.com/9877852/32395547-cfcd7dd8-c09e-11e7-9cd3-5e86ed34075b.png)
^ just showing the space difference